app/views/admin/workshops/_invitation_management.html.haml

```diff
@@ -25,15 +25,15 @@
     .medium-6.columns
       %h4
         Students
-        %label.label.round.secondary #{@attending_students.count}#{(" of " + @workshop.host.seats.to_s) if @workshop.has_host?}
+        %label.label.round.secondary #{@attending_students.count} of #{@workshop.student_spaces}
         %label.label.round.signed-in #{ @attending_students.count(&:attended) }
       = render partial: 'admin/workshop/attendances', locals: { invitations: @attending_students }
       = render partial: 'admin/workshop/waiting_list', locals: { waiting_list: @student_waiting_list }
     .medium-6.columns
       .row
         %h4
           Coaches
-          %label.label.round.secondary #{@attending_coaches.count}#{(" of " + @workshop.host.coach_spots.to_s) if @workshop.has_host?}
+          %label.label.round.secondary #{@attending_coaches.count} of #{@workshop.coach_spaces}
```

Comment:
I'm not sure why we need a variable for `@attending_coaches`. Investigate and remove.

Reply:
This is a decorated collection and used elsewhere, so I'd rather not change it on this PR.